Source/AwsCommonRuntimeKit/auth/credentials/CRTAWSCredentialsProvider.swift

```diff
@@ -18,11 +18,13 @@ public final class CRTAWSCredentialsProvider {
     }
 
     public convenience init(fromProvider impl: CRTCredentialsProvider,
-                            shutDownOptions: CRTCredentialsProviderShutdownOptions? = nil,
-                            allocator: Allocator = defaultAllocator) throws {
-        let shutDownOptions = CRTAWSCredentialsProvider.setUpShutDownOptions(shutDownOptions: shutDownOptions)
+                            allocator: Allocator = defaultAllocator,
+                            shutdownCallback: ShutdownCallback? = nil) throws {
```

Comment:
3 points to Gryffindor for renaming ~shutDown~ -> `shutdown`

Reply:
🎉